src/coretemp.c

```diff
@@ -54,152 +54,148 @@ static const char *config_keys[] = {
-
-//global config vars :)
-int MaxValues = 0;
-int ValuesPercentage = 0;
-int ValuesDegrees = 1;
+// global config vars :)
+int             MaxValues = 0;
+int             ValuesPercentage = 0;
+int             ValuesDegrees = 1;
 
 struct coretemp_core
 {
-  unsigned int tjmax;
-  unsigned int socket;
-  unsigned int core;
-  unsigned int hwmon;
-  int input;
-  char *label;
+  unsigned int    tjmax;
+  unsigned int    socket;
+  unsigned int    core;
+  unsigned int    hwmon;
+  int             input;
+  char           *label;
 };
 
 struct coretemp_core **c = 0;
-unsigned int core_count = 0;
+unsigned int    core_count = 0;
 
 // free the coretemp_core struct list
-static int
-coretemp_cleanup ()
+static int coretemp_cleanup()
 {
-  int i;
+  int             i;
   for (i = 0; i < core_count; i++)
   {
     if (c[i]->input)
-      close (c[i]->input);
+      close(c[i]->input);
     if (c[i]->label)
-      free (c[i]->label);
-    free (c[i]);
+      free(c[i]->label);
+    free(c[i]);
   }
   if (c)
-    free (c);
+    free(c);
   c = 0;
 
   return 0;
 }
 
-static int
-coretemp_findcores ()
+static int coretemp_findcores()
 {
-  DIR *dp = 0, *dp2 = 0, *dp3 = 0, *dp4 = 0;
-  struct dirent *de, *de2, *de3, *de4;
-  unsigned int socket = 0;
-  int l, core, idx, i, hwmon = 0;
-  char core_name[100];
-  char *s, *b, t;
-  void *realloc_sucks;
-  FILE *f = 0;
-
-  if (chdir (CORETEMP_PATH))
+  DIR            *dp = 0, *dp2 = 0, *dp3 = 0, *dp4 = 0;
+  struct dirent  *de, *de2, *de3, *de4;
+  unsigned int    socket = 0;
+  int             l, core, idx, i, hwmon = 0;
+  char            core_name[100];
+  char           *s, *b, t;
+  void           *realloc_sucks;
+  FILE           *f = 0;
+
+  if (chdir(CORETEMP_PATH))
   {
-    ERROR ("coretemp plugin: unable to find coretemp path (%s)",
-	   CORETEMP_PATH);
+    ERROR("coretemp plugin: unable to find coretemp path (%s)",
+	  CORETEMP_PATH);
     return (-1);
   }
 
-  if (!(dp = opendir (".")))
+  if (!(dp = opendir(".")))
   {
-    ERROR ("coretemp plugin: unable to diropen() coretemp path (%s)",
-	   CORETEMP_PATH);
+    ERROR("coretemp plugin: unable to diropen() coretemp path (%s)",
+	  CORETEMP_PATH);
     return (-1);
   }
 
-  while ((de = readdir (dp)))
+  while ((de = readdir(dp)))
   {
     if (de->d_name[0] == '.')
       continue;
-    if (!strncmp (de->d_name, CORETEMP_NAME, CORETEMP_STRLEN))
+    if (!strncmp(de->d_name, CORETEMP_NAME, CORETEMP_STRLEN))
     {
-      socket = atoi (&de->d_name[CORETEMP_STRLEN]);
-      if (chdir (de->d_name))
+      socket = atoi(&de->d_name[CORETEMP_STRLEN]);
+      if (chdir(de->d_name))
       {
 	ERROR
-	  ("coretemp plugin: unable to chdir into [ %s ] for coretemp data",
-	   de->d_name);
+	    ("coretemp plugin: unable to chdir into [ %s ] for coretemp data",
+	     de->d_name);
 	goto error_exit;
       }
-      dp2 = opendir (".");
-      int hwmon_mode = 1;
-      while ((de2 = readdir (dp2)))
+      dp2 = opendir(".");
```

Comment:
our existing style is a space between a function name and the braces. Please keep that.